packages/fx-core/src/common/samples.ts

```diff
@@ -37,34 +42,42 @@ interface SampleCollection {
 class SampleProvider {
   private sampleCollection: SampleCollection | undefined;
   private samplesConfig: { samples: Array<Record<string, unknown>> } | undefined;
+  private branchOrTag = SampleConfigTag;
 
   public async fetchSampleConfig() {
-    try {
-      const fileResponse = await sendRequestWithTimeout(
-        async () => {
-          return await axios.get(
-            `https://raw.githubusercontent.com/${SampleConfigOwner}/${SampleConfigRepo}/${SampleConfigTag}/${SampleConfigFile}`,
-            { responseType: "json" }
-          );
-        },
-        1000,
-        3
-      );
-
-      if (fileResponse && fileResponse.data) {
-        this.samplesConfig = fileResponse.data as { samples: Array<Record<string, unknown>> };
+    const version: string = packageJson.version;
+    if (version.includes("alpha")) {
+      // daily build version always use 'dev' branch
+      this.branchOrTag = "dev";
+    } else if (version.includes("rc")) {
+      // rc version always use prerelease tag
+      this.branchOrTag = SampleConfigTagForPrerelease;
```

Comment:
I suppose rc version is prerelease version before stable version, we should use v3 for testing?

Reply:
This is designed for breaking changes in future. If we need to release sample changes now, I think we should test the samples using the stable version TTK and set feature flag to 'v3' instead of using prerelease TTK.